src/main.rs

```diff
@@ -45,6 +53,27 @@ mod options;
 mod output;
 mod theme;
 
+lazy_static! {
```

Comment:
I would like this to have a comment explaining what it does.

Reply:
```suggestion
/// A lazily initialized static map of all mounted file systems.
///
/// This map contains a mapping from the mount destination (`PathBuf`) to the corresponding `MountedFs` struct.
/// On Linux systems, this map is populated using the `MountList` struct. If there's an error retrieving the mount list or
/// if the target OS is not Linux, the map will be initialized as empty.
///
/// # `MountedFs` struct
///
/// Each `MountedFs` struct contains:
/// - `dest`: The destination path of the mount as a string.
/// - `fstype`: The file system type.
/// - `source`: The source of the mount as a string.
///
/// # Examples
///
/// Accessing a mounted file system's details:
///
/// ```
/// let maybe_mount = ALL_MOUNTS.get(Path::new("/some/mount/point"));
/// if let Some(mount) = maybe_mount {
///     println!("File system type: {}", mount.fstype);
/// }
/// ```
lazy_static! {
```

I asked chatgpt to generate the comment, it might be useful as a template.